README.md

```diff
@@ -1,28 +1,49 @@
-# CHAOSS Common Metrics Working Group (Common WG) 
+# CHAOSS Common Metrics Working Group
 
-## Join our Mailing List
+## Table of Contents
 
-This CHAOSS working group is using the general [CHAOSS mailing list](https://lists.linuxfoundation.org/mailman/listinfo/chaoss).
+- [Introduction](#introduction)
+- [Participate](#participate)
+- [Contributing](#contributing)
+- [Metrics](#metrics)
+- [Contributors](#contributors)
+- [License](#license)
 
-## Join our Weekly Calls
+## Introduction
 
-The Common Metrics WG meets every other Thursday at 10:00am CT (usually 17:00 CET, [check your local time](https://arewemeetingyet.com/Chicago/2020-11-12/10:00/b/CHAOSS%20Common%20Metrics%20WG#eyJ1cmwiOiJodHRwczovL3pvb20udXMvai80OTk4Njg3NTMzIn0=)) via [Zoom](https://zoom.us/j/4998687533). All are welcome!
+### Goals 
 
-For more details, you can read our [agenda and meeting minutes document](https://bit.ly/2ROytFz)
+The Common Metrics Working Group focuses on defining the metrics that are used by both working groups or are important for community health.
```

Comment:
Can you change "both" to "multiple"?

Reply:
Thanks for the suggestion! I changed it.

Reply:
Thank you for all of your help with this!